src/components/TwoPhotosSection/TwoPhotosSection.js

```diff
@@ -1,8 +1,38 @@
-import React from "react";
+import React, { useRef, useEffect, useState } from "react";
```

Comment:
Some extra feedback on the project:
- The layout is fullwidth - in bigger resolutions, it's too big and not very readable. it would be nice to set up some width limits. (maybe around 1000px)
- Review the page headings structure. Every page must have one h1 and have [a linear heading order](https://web.dev/heading-order/).
- The gray color for text is too light, with low contrast ratio.
- Well done adding alts where need, but they can be improved. eg "go to facebook page", can be just "facebook". the machines will know that it's an image inside a link, so it will announce it properly. 
- I noticed you used `em` for margins and padding, did you meant `rem`?

Reply:
For heading order and text in general I did find it hard to decide what makes sense, a lot of things ended up in ```<p>```'s as the result. 

As for em's and rem's, I realize now I probably should have given it more thought. It doesn't help that there are so many different opinions on this topic (like <a href='https://blog.usejournal.com/dont-use-rem-em-for-paddings-margins-and-more-94e19026b000'>this one</a> that says use relative units exclusively for font size). What do you suggest as the best approach?

Reply:
@majac91 it's okay if a lot of things are just `<p>` as long as the h1 is there and the following headings are in the correct order.

For em/rem, the article you mentioned has a strong point. I found about it a year ago and even I have been rethinking the way I approach it. Nowadays I do pretty much that: Use relative units exclusively for text. It's a hot topic with strong opinions on both sides. Here's a [Twitter thread](https://twitter.com/a_sandrina_p/status/1258133389523259400) about the topic